cmdlineutils.adoc

```diff
@@ -2309,9 +2309,13 @@ zmrestore -a user1@domain.com -ca -pre new_
 
 === zmrestoreoffline (Offline Restore)
 
-Run `zmrestoreoffline` when the Zimbra server (i.e., the mailbox process)
-is down.  The SQL database for the server and the OpenLDAP directory
-server must be running before you start the `zmrestoreoffline` command.
+`zmrestoreoffline` requires that the following is true:
+
+- mailboxd IS NOT running
+- SQL database IS RUNNING
+- OpenLDAP directory IS RUNNING
```

Comment:
nit: recommend changing OpenLDAP to just LDAP (in this doc we're usually, but not always just referring to our LDAP server without specifically stating OpenLDAP)

Reply:
I was attempting to copy the usage in this document itself.  Should all instances of OpenLDAP be changed then?

Reply:
My quick check of this file shows (with this change) 2 instances of "OpenLDAP" and 49 instances of " ldap" (case insensitive).  I'd change the one other instance of OpenLDAP beside this one to something more consistent with the rest of the doc.

Reply:
Roger that.  When I added that I found the closest mention of LDAP and copied it's usage.  I'll be more thorough in the future as far as inspecting the actual usage in the document.